server/src/server.ts

```diff
@@ -148,6 +153,37 @@ disposables.push(
     analyzer.clearRecipeLocalFiles()
   }),
 
+  connection.onCodeLens(async (params): Promise<LSP.CodeLens[]> => {

```

Comment:
Maybe you'd like to put this into a `onCodeLens` file in `connectionHandlers`, but this works too

Reply:
Yes, like we do other feature handlers, it would be nice to have a dedicated file for separating features.